_source/logzio_collections/_prometheus-sources/otel-metrics-k8s-helm.md

```diff
@@ -0,0 +1,103 @@
+{:.info-box.note}
+<!-- info-box-end -->
+
+#### Standard configuration
+
+<div class="tasklist">
+
+##### Deploy
+
+###### Parameter configuration
+
+Replace the Logz-io `<<PROMETHEUS-METRICS-SHIPPING-TOKEN>>` with the [token](https://app.logz.io/#/dashboard/settings/manage-tokens/data-shipping) of the metrics account to which you want to send your data.
+
+
+Replace `<<LISTENER-HOST>>` with your region’s listener host (for example, `https://listener.logz.io:8053`). For more information on finding your account’s region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html).
+
+Replace `<<ENV-TAG>>` with the name for the environment's metrics, to easily identify the metrics for each environment.
+
+###### Deployment code
+
+```
+helm install  \
+--set secrets.MetricsToken=<<PROMETHEUS-METRICS-SHIPPING-TOKEN>> \
+--set secrets.ListenerHost=<<LISTENER-HOST>> \
+--set secrets.p8s_logzio_name=<<ENV-TAG>> \
+logzio-otel-k8s-metrics logzio-otel/logzio-otel-k8s-metrics
+```
+
+##### Check Logz.io for your metrics
+
+Give your metrics some time to get from your system to ours, and then open [Logz.io](https://app.logz.io/).
+
+
+####  Customizing default parameters
+
+You can use the following options to update your default parameter values: 
+
+* Specify parameters using the `--set key=value[,key=value]` argument to `helm install`
+
+* Edit the `values.yaml`
+
+* Overide default values with your own `my_values.yaml` and apply it in the `helm install` command. 
+
+###### Example:
+
+```
+helm install logzio-otel-k8s-metrics logzio-otel/logzio-otel-k8s-metrics -f my_values.yaml 
+```
+
+#### Customizing which metrics are collected  
+
+The default configuration uses the Prometheus receiver with the following scrape jobs:
+
+* Cadvisor: Scrapes container metrics
+* Kubernetes service endpoints: These jobs scrape metrics from the node exporters, from Kube state metrics, from any other service for which the `prometheus.io/scrape: true` annotaion is set, and from services that expose Prometheus metrics at the `/metrics` endpoint.
+##### Customize the metrics collection
+
+To customize your configuration, edit the `config` section in the `values.yaml` file.
+
+#### Uninstalling the Chart
+
+The uninstall command is used to remove all the Kubernetes components associated with the chart and to delete the release.  
+
+##### To uninstall the `logzio-otel-k8s-metrics` deployment
+
+Use the following command:
+
+```shell
+helm uninstall logzio-otel-k8s-metrics
+```
+</div>
+
+### Change log
```

Comment:
Where is the content for this section? 
How is it related to the previous sections?

Reply:
I think we can remove this section from our docs it is more relevant for the Github repo